titus-client/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/GrpcJobManagementModelConverters.java

```diff
@@ -97,6 +76,8 @@
 import com.netflix.titus.grpc.protogen.SecurityProfile;
 import com.netflix.titus.grpc.protogen.ServiceJobSpec;
 import com.netflix.titus.grpc.protogen.SignedAddressAllocation;
+import io.kubernetes.client.openapi.models.V1ContainerStatus;
+import io.kubernetes.client.proto.V1;
```

Comment:
Can you confirm this is intentional? Sometimes things auto-add imports but not quite the right "containerstatus".
We probably don't want this file to "know" about k8s at this point.

Reply:
it probably pulled in the wrong one, ill clean it up